### DISCUSSION TITLE
feeeback app

### DISCUSSION THREAD

**Discussion by itsmmt305** [libre-tube/LibreTube#3717] [Q&A]
>frivolous

third party companies manufacturing hardware and skins for android have a lot of bullshits. bugs, restrictions, poor layout designs, indecent and ugly ui, inattention to detail. their apps being mostly closed source disable users to either create or procure fixes for such issues. this leads to amped up anger and frustration in such individuals which goes unheaded.
so would y'all be interested in a foss app created by bnyro made with material ui 3 (you) that hosts a platform for such users to openly and brutally criticize the companies propogating such nonsense? this would privide a vent for their agitation and probably even help their case. what would you like it to be called considering the nomenclature bnyro uses?

**Comment by Bnyro** [comment]
If you want to complain about how bad other closed source apps or companies are, you can do so at https://matrix.to/#/!RzYjpjhvNbGMLHqmFM:matrix.org?via=matrix.org&via=projectsegfau.lt&via=tchncs.de
